content/guides/servicediscovery.md

```diff
@@ -139,6 +139,12 @@ example:
     docker run -d --name dd-agent -h `hostname` -v /var/run/docker.sock:/var/run/docker.sock -v /proc/:/host/proc/:ro -v /sys/fs/cgroup/:/host/sys/fs/cgroup:ro -e API_KEY=[YOUR_API_KEY] -e SD_CONFIG_BACKEND=etcd -e SD_BACKEND=docker -e SD_BACKEND_HOST=[YOUR_ETCD_IP] -e SD_BACKEND_PORT=[YOUR_ETCD_PORT] datadog/docker-dd-agent:kubernetes
 
 
+### Using configuration files instead of a configuration store
+If running a configuration store is not possible in your environment but shipping configuration files with the agent is, you can use the [conf.d/auto_conf folder](https://github.com/DataDog/dd-agent/tree/master/conf.d/auto_conf) to store configuration templates. The format is pretty simple and looks like the typical YAML configuration file for checks with just one more field.
+Use existing files in this folder as an example.
+If you use this instead of a K/V store you still need to uncomment `service_discovery_backend: docker` in `datadog.conf`, but `sd_config_backend`, `sd_backend_host` and `sd_backend_port` can be omitted.
```

Comment:
Let's change "can be omitted" to "must be omitted", since setting an `sd_config_backend` sets auto_conf to false and the agent wont load the templates.